server/routers/interview.ts

```diff
@@ -118,6 +118,26 @@ export const interviewRouter = router({
 
         return interview;
       }),
+    manyByIds: protectedProcedure
+      .input(
+        z.array(
+          z.object({
+            id: z.string(),
+          }),
+        ),
+      )
+      .query(async ({ input: data }) => {
+        const interviewIds = data.map((p) => p.id);
+        const interviews = await prisma.interview.findMany({
+          where: {
+            id: {
+              in: interviewIds,
+            },
+          },
+        });
```

Comment:
If you include the protocol here, you don't need a separate query to fetch Protocols in the action.